sugoi-api-core/src/main/java/fr/insee/sugoi/core/service/impl/GroupServiceImpl.java

```diff
@@ -205,98 +186,85 @@ public void update(String realm, String appName, Group group) {
               Map.entry(EventKeysConfig.GROUP, group),
               Map.entry(EventKeysConfig.APPLICATION_NAME, appName),
               Map.entry(EventKeysConfig.ERROR, e.toString())));
-      if (e instanceof GroupNotFoundException) {
-        throw (GroupNotFoundException) e;
-      } else {
-        throw e;
-      }
+      throw e;
     }
   }
 
+  // TODO attention il manque le userStorage du userId a ajouter
   @Override
-  public void addUserToGroup(String realm, String userId, String appName, String groupName) {
+  public ProviderResponse addUserToGroup(
+      String realm,
+      String storage,
```

Comment:
userStorage ?

Reply:
why ?

Reply:
We need the userstorage at this level because we need to know where to find the user, indeed adding a user to a group in ldap require to know on which userstorage he is because the user dn depends on the userstorage